front/app/components/ProjectAndFolderCards/ProjectAndFolderCards.test.tsx

```diff
@@ -341,42 +322,6 @@ describe('<ProjectAndFolderCards />', () => {
     expect(filterSelector).toBeInTheDocument();
   });
 
-  it('calls onChangeAreas on change areas', () => {
```

Comment:
Why did you delete all these tests?

Reply:
Because it's working a little differently with the new hooks and I was under the impression that this is covered by e2e tests but it wasn't exactly the case. I updated them and brought them back 👍